Tests/Form/Type/AssertFormReferenceCompatibleWithSymfony28Trait.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\MediaBundle\Tests\Form\Type;
+
+trait AssertFormReferenceCompatibleWithSymfony28Trait
```

Comment:
Should be okay, because the trait is located under `Tests` which shouldn't be added to the (non-dev) classloader.